src/json.rs

```diff
@@ -24,7 +24,6 @@ struct Diagnostic {
     level: String,
     spans: Vec<DiagnosticSpan>,
     children: Vec<Diagnostic>,
-    rendered: Option<String>,
```

Comment:
This just fixes the build warning, right? Could I have that in a separate PR?

Reply:
Sure.

Reply:
https://github.com/laumann/compiletest-rs/pull/108

Reply:
Thanks